Tools/TestWebKitAPI/Tests/WebKitGLib/TestDownloads.cpp

```diff
@@ -588,17 +588,30 @@ class WebViewDownloadTest: public WebViewTest {
 
     void waitUntilDownloadStarted()
     {
-        m_download = 0;
+        m_download = nullptr;
         g_main_loop_run(m_mainLoop);
         g_assert_nonnull(m_download.get());
     }
 
+    static void decideDestination(WebViewDownloadTest* test)
+    {
+        GUniquePtr<char> destination(g_build_filename(Test::dataDirectory(), test->m_suggestedFilename.data(), nullptr));
+        webkit_download_set_destination(test->m_download.get(), destination.get());
+    }
+
```

Comment:
Why is this static? it uses m_suggstedFilename and m_download

Reply:
There is a virtual member function with the same name. Probably too confusing. I'll just get rid of it.

Reply:
Decided to rename it, actually.